frontend/packages/core/src/AppLayout/header.tsx

```diff
@@ -14,6 +14,11 @@ import { UserInformation } from "./user";
 
 export const APP_BAR_HEIGHT = "64px";
 
+export interface HeaderProps {
+  title?: string;
+  logo?: React.ReactNode;
+}
```

Comment:
should the appconfiguration interface be re-used or vice/versa in case new props are added in 1 place but forgotten to be added in the other interface?

Reply:
Probably makes sense, I updated it to reuse the AppConfiguration in the header now